### DISCUSSION TITLE
`Santos` vs multiple `San`

### DISCUSSION THREAD

**Discussion by tukusejssirs** [romcal/romcal#452] [Ideas]
> [from a [comment](https://github.com/romcal/romcal/pull/431#issuecomment-1696312719) by @emagnier]
>
> On liturgical books in French (as I am French), this is also similar: in the Missal, the equivalent "San" is also placed before each name, whereas in the Lectionary, the equivalent "Santos" is used. This could possibly be something to consider for standardizing within Romcal in the future...

Just a small comment on this `Santos` vs multiple `San`: IMHO we should always prefer the version used by Roman Missal in a particular locale, unless there is a much newer (e.g. 5-10 years?) official publication (e.g. lectionary, the Liturgy of the Hours) available. When there is no official Roman Missal translation available (to us), we should use the most official (first > second > thirdy party) translation available to us.

**Comment by emagnier** [comment]
That works for me too. I'm adding the `Documentation` label to remember this when we have the opportunity to rework the contribution guide.
